### ISSUE TITLE
My Teams should be part of the home page

### ISSUE THREAD

**Issue by RichTillis** [RichTillis/rsl-az-soccer#14]
When I was using the app, 90% of the time it was to get information on the teams I had favorited. So every time I fired up the app I had to navigate my favorites list and then click my team.  It would be great to have the my-team listing on the home page.

**Comment by joshshaloo** [comment]
As a tournament official, we want them seeing the activity stream. One change we can do is make a my schedule page that shows a consolidated schedule of all the teams you follow.

**Comment by RichTillis** [comment]
role-based home page maybe? I would think a majority or our users will be parents/players/family etc. Role-based would support both of these (and other) types of users

**Comment by joshshaloo** [comment]
Right, we want parents/players/family etc seeing the activity stream.
